src/go/k8s/cmd/main.go

```diff
@@ -185,6 +181,7 @@ func main() {
 	flag.StringSliceVar(&additionalControllers, "additional-controllers", []string{""}, fmt.Sprintf("which controllers to run, available: all, %s", strings.Join(availableControllers, ", ")))
 	flag.BoolVar(&operatorMode, "operator-mode", true, "enables to run as an operator, setting this to false will disable cluster (deprecated), redpanda resources reconciliation.")
 	flag.BoolVar(&enableHelmControllers, "enable-helm-controllers", true, "if a namespace is defined and operator mode is true, this enables the use of helm controllers to manage fluxcd helm resources.")
+	flag.DurationVar(&unbindPVCsAfter, "unbind-pvcs-after", 0, "if not zero, runs the PVCUnbinder controller which attempts to 'unbind' the PVCs' of Pods that are Pending for longer than the given duration")
```

Comment:
I like that it only does unbinding if the pod is pending for a giving amount of time.

Reply:
is it possible to tell it not to unbind PVCs if the majority of redpanda pods are in pending state? and instead page a human?

Reply:
This is copied from our slack conversation:

> is it possible to tell it not to unbind PVCs if the majority of redpanda pods (quroum) are in pending state?

This is the case I don't have a solution for. Though I don't think we need to. If all the Pods go into a Pending state, a human should be getting paged anyways. The controller will unbind all Pods but leave the PVs around.

The worst case here is that a completely new redpanda cluster is created. That would only happen if every redpanda Pod is rolled at the same time and the scheduler has a bug (which we have seen). The controller would unbind all PVs and delete the PVCs. Then Kubernetes would make all new PVs because none of the existing ones could be used due to the scheduler bug.

If Pods were rolled one at a time, it's just a race about if data can up replicate in time. Though Pods being unavailable should stop any updates unless something really bad is happening.